src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocIncorrect.java

```diff
@@ -122,31 +122,38 @@ void foo5() {}
          * An especially short bit of Javadoc.
          */
         void foo6() {}
-    // violation below
+    // violation below 'First sentence .* missing an ending period.'
          /**
           * mm{@inheritDoc}
           */
          void foo7() {}
-    // violation below
+    // violation below 'Summary javadoc is missing.'
          /**
           * {@link #setBounds(int,int,int,int)}
           */
          void foo8() {}
-    // violation below
+    // violation below 'Summary javadoc is missing.'
          /**
           *
           */
          void foo10() {}
     };
-    // violation below
+    // violation below 'First sentence .* missing an ending period.'
     /**
      * M m m m {@inheritDoc}
      */
     void foo7() {}
-    // violation below
+    // violation below 'Summary javadoc is missing.'
     /** */
     <T> T foo8(T t) {return null;}
-    // violation below
+    // violation below 'Summary javadoc is missing.'
     /** */
     String[] foo9() {return null;}
+
+    /**
+     * This is the summary.
+     * {@summary This is NOT summary according to javadoc tool}
+     */ // violation above 'Summary .* missing an ending period.'
```

Comment:
Line 154 has period at the end. Why we have this violation ?

Reply:
first line in javadoc is ignored, it is not summary. Only summary tag content is considered and therefore checked

Reply:
```
     * This is the summary.
     * {@summary This is NOT summary according to javadoc tool}
```

![image](https://user-images.githubusercontent.com/812984/147393069-82c73f08-19f3-4dc2-beab-e68e7afa9301.png)

Am I miss something ?

Reply:
@romani No, you are right, if the first sentence is present then the `@summary` is ignored, I mentioned it [here](https://github.com/checkstyle/checkstyle/issues/11030#issuecomment-994288137) but the violation above is not for an actual summary but for `@summary`.

```
    /**
     * This is the summary.
     * {@summary This is NOT summary according to javadoc tool}
     */ // violation above 'Summary .* missing an ending period.'
     */
```
There are two messages for missing ending period
```
summary.first.sentence=First sentence of Javadoc is missing an ending period.
summary.javaDoc.missing.period=Summary of Javadoc is missing an ending period.
```
This is a bug, if normal summary (without using `@summary`)  is present then `@summary` should be ignored though no one would write such a javadoc. Just mentioning, this is not due to this PR but is existing behaviour, though the fix won't be difficult.
One more thing, I feel that these two messages are very confusing and there should be only one message, same for with `@summary` and without it.

Reply:
@romani Please share your opinion.

Reply:
if such problem is not introduced in this PR, please create new issue on this , we can fix is separate PR.

Reply:
it would be awesome to make comment in  Input like `// violation above 'Summary .* missing an ending period.' until #Xxxxx`

Reply:
Created issue at https://github.com/checkstyle/checkstyle/issues/11213